lesson02/launch_confg.tf

```diff
@@ -13,3 +9,4 @@ variable "associate_public_ip_address" {
 variable "iam_role" {
   default = ""
 }
+
```

Comment:
styling: let's remove the extra line

Reply:
we can do this in the next cycle this is not a blocker, thanks for pointing that out

Reply:
This will be done in another PR. Created an issue to keep track of this #33